Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM mesosphere/mesos:0.21.1-1.1.ubuntu1404
+
+ENV HDFS_MESOS_VERSION 0.0.2
```

Comment:
In docker it's a good idea to bind this env variable as late as possible, this is because running this directive will generate a new layer, which will invalidate any layer that was created after this directive.  In order to allow dockers layer caching to work as effectively as possible things should be bound as late as possible.